### PR TITLE
Support GHC 8.10

### PR DESCRIPTION
**Pull request by sjakobi**

### CODE REVIEW

File: tasty-expected-failure.cabal [hunk 1]
```diff
@@ -27,13 +27,13 @@ category:            Testing
...
 cabal-version:       >=1.10
-tested-with: GHC == 7.4.2, GHC == 7.6.3, GHC == 7.8.4, GHC == 7.10.3, GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.4, GHC == 8.6.5, GHC == 8.8.1
+tested-with: GHC == 7.4.2, GHC == 7.6.3, GHC == 7.8.4, GHC == 7.10.3, GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.4, GHC == 8.6.5, GHC == 8.8.3, GHC == 8.10.1
 
 library
   exposed-modules:
     Test.Tasty.ExpectedFailure
   build-depends:
-    base >= 4.5 && <4.14,
+    base >= 4.5 && <5,
```
Comment (sjakobi) [inline tasty-expected-failure.cabal:36]: My impression of the code in this package is that it is sufficiently simple that it won't require any changes soon.

**Comment by sjakobi** [comment]
Since there seems to be an issue with Travis' GitHub integration, here's the build job for this PR:

https://travis-ci.org/github/nomeata/tasty-expected-failure/builds/668787094

**Review by nomeata — APPROVED** [review]

**Comment by nomeata** [comment]
Thanks! I updated hackage metadata of existing package (easier than uploading)

**Comment by sjakobi** [comment]
Thank you! :)

**Pull request merged**